src/components/Select.jsx

```diff
@@ -355,18 +353,13 @@ export default class Select extends Component {
       const menuNode = React.findDOMNode(this.refs.menu);
 
       // the menu was just opened
-      if (!previousState.isOpen && this.state.isOpen) {
+      if (!previousState.isOpen && this.state.isOpen && this.props.children && this.props.children.length > 0) {
         const positionOptions = has(this.props, 'positionOptions') ? this.props.positionOptions : config.positionOptions;
         positionOptions(this);
       // restore the old scrollTop position
       } else {
         menuNode.scrollTop = this.cachedMenuScrollTop;
       }
-
-      if (!previousState.isOpen && this.state.isOpen) {
```

Comment:
I have actually changed another line 704 for this, this was creating issue when select has no options and ```<ul>``` was getting styled ```{display: block}```. Thats was creating a small issue.

If you think its needed here I can add condition to this if to check if select has options.